### PR TITLE
fix: broken link

### PR DESCRIPTION
**Pull request by tpoindessous**
##### SUMMARY
<!— Your description here –>
Just a fix for a broken link.
right page is now on : https://docs.ansible.com/ansible/latest/reference_appendices/playbooks_keywords.html#task

Thanks

##### ISSUE TYPE
- Docs Pull Request

+label: docsite_pr

##### ISSUE TYPE

- Bugfix Pull Request
- Docs Pull Request
- Feature Pull Request
- Test Pull Request

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

File: lib/ansible/modules/command.py [hunk 1]
```diff
@@ -21,7 +21,7 @@
        like C("*"), C("<"), C(">"), C("|"), C(";") and C("&") will not work.
        Use the M(ansible.builtin.shell) module if you need these features.
      - To create C(command) tasks that are easier to read than the ones using space-delimited
-       arguments, pass parameters using the C(args) L(task keyword,../reference_appendices/playbooks_keywords.html#task)
+       arguments, pass parameters using the C(args) L(task keyword,../../../reference_appendices/playbooks_keywords.html#task)
```
Comment (bcoca) [inline lib/ansible/modules/command.py]: this should not be a relative link at all, as rendering in any context outside of docs.ansible.com will create a broken link (i.e galaxy.ansible.com)

**Review by bcoca — CHANGES_REQUESTED** [review]

File: lib/ansible/modules/command.py (hunk 1 above)
Comment (tpoindessous) [reply lib/ansible/modules/command.py]: Thanks ! no luck for me, I tried to correct the only link which wasn't correct 😄

**Comment by samccann** [comment]
Hi @tpoindessous and thanks for helping to fix the broken links!

I don't know the right answer, but changing this to an absolute link has its own problems. If we move that page again, that will become a new broken link, for example.

This may be complex in that we have three different spaces that use this output:

- docsite - which uses anchors and the R() format.
- ansible-doc output - which doesn't seem to recognize that R() format
- GalaxyNG, etc - not sure yet what they recognize, out of all the options available.

Note this use of L() was something we stopped using a few releases ago, but didn't go into the existing links to see where it was used and how to replace it.  Which is my way of saying sorry, this simple PR might take a little bit to resolve as we figure out what the correct approach is.

**Review by samccann — APPROVED** [review]

**Comment by samccann** [comment]
Related to https://github.com/ansible-community/community-team/issues/103... Merging because it's an improvement over the broken link for sure!  Thanks @tpoindessous for  the ansible Docs fix and welcome to the Ansible project!

**Pull request merged**